app/assets/stylesheets/vendor/_select2.sass

```diff
@@ -0,0 +1,422 @@
+.select2-container
```

Comment:
OK.
I don't know where to find <head> of app where css files (bootstrap, application, select) are loaded.
Furthermore I'm not sure how sass compilation works right now - it seems only application.sass is compiled - thus also this file (select3.sass) would have to be added to compilation as well or it can be compiled manually once (if it's more like a lib).